indexing-service/src/main/java/org/apache/druid/indexing/common/actions/SegmentLockAquireAction.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.indexing.common.actions;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.google.common.base.Preconditions;
+import org.apache.druid.indexing.common.TaskLockType;
+import org.apache.druid.indexing.common.task.Task;
+import org.apache.druid.indexing.overlord.LockResult;
+import org.apache.druid.indexing.overlord.SpecificSegmentLockRequest;
+import org.joda.time.Interval;
+
+/**
+ * TaskAction to acquire a task lock for a segment. Used by stream ingestion tasks.
+ */
+public class SegmentLockAquireAction implements TaskAction<LockResult>
```

Comment:
`SegmentLockAquireAction` -> `SegmentLockAcquireAction`

Reply:
Can this class be combined with `SegmentLockTryAcquireAction`?

Reply:
Thanks, fixed name. 

> Can this class be combined with SegmentLockTryAcquireAction?

 I prefer to not combine them since it's clearer in task logs what the task is doing.